### DISCUSSION TITLE
Thank you for implementing JQL Queries

### DISCUSSION THREAD

**Discussion by sunnythaper** [adyscorpius/logseq-jira#16] [General]
I cannot stress how much I've been wanting this feature! I'm a fan of the implementation. I was just wondering if multiple JQL queries was on the horizon?

**Comment by adyscorpius** [comment]
Hey, thanks for sharing that. I did think of extending this capability, especially multiple JQLs. 

How do you plan to use it? What are some queries you'd like to run?

**Reply by sunnythaper** [reply]
Aside from listing out my own tasks, I manage a team of 4 designers at my job and often need to list out their individual tasks as well. That's where I'm thinking multiple JQLs may help me the most. 

The other thing I am not 100% sure I would use but the idea of it is definitely sounding appealing to me now is being able to list out currently active epics within Logseq.

I also span a few projects at work and may want to use JQLs to get tasks specific to each project I am on as well.

Hope this helps spur some thoughts!

**Reply by sunnythaper** [reply]
I have a new use case. I was thinking of  creating a weekly report page in Logseq that I can update each Monday and send my team a copy. It would be great if that could be the following format with multiple JQLs:

- **Active Epics**
  - `JQL QUERY`
- **Open Tasks This Sprint**
  - Sunny (me)
    - `JQL QUERY`
  - Designer 1
    - `JQL QUERY`
  - Designer 2
    - `JQL QUERY`
  - Designer 3
    - `JQL QUERY`
  - Designer 4
    - `JQL QUERY`
- **Completed Tasks This Sprint**
  - `JQL QUERY`
- **Written Notes**
  - Such as goals for the week, etc.

**Reply by adyscorpius** [reply]
Those are great use cases.. I think this sounds like a great code block to implement. Let me explore how it works.
